From c71f8e54f5ceb97f4ebd6b26127d41639ab0e351 Mon Sep 17 00:00:00 2001
From: lijian <jian.li@freescale.com>
Date: Thu, 29 Dec 2011 06:12:57 +0800
Subject: [PATCH] ENGR00133115 Add .ts,.vob,.f4v,.mkv,.webm mimetype

Add these mimetype to make these files to http
streaming not download source.

Signed-off-by: lijian <jian.li@freescale.com>
---
 luni/src/main/java/libcore/net/MimeUtils.java | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/luni/src/main/java/libcore/net/MimeUtils.java b/luni/src/main/java/libcore/net/MimeUtils.java
index aaa5670..ac786f8 100644
--- a/luni/src/main/java/libcore/net/MimeUtils.java
+++ b/luni/src/main/java/libcore/net/MimeUtils.java
@@ -362,6 +362,11 @@ public final class MimeUtils {
         add("video/x-webex", "wrf");
         add("x-conference/x-cooltalk", "ice");
         add("x-epoc/x-sisx-app", "sisx");
+        add("video/mpeg", "vob");
+        add("video/mpeg", "ts");
+        add("video/x-flv", "f4v");
+        add("video/x-matroska", "mkv");
+        add("video/x-matroska", "webm");
         applyOverrides();
     }
 
-- 
1.8.0

